op-service/txmgr/txmgr.go

```diff
@@ -589,6 +584,25 @@ func (m *SimpleTxManager) suggestGasPriceCaps(ctx context.Context) (*big.Int, *b
 	return tip, head.BaseFee, nil
 }
 
+func (m *SimpleTxManager) checkLimits(tip, basefee, bumpedTip, bumpedFee *big.Int) error {
+	// If below threshold, don't apply multiplier limit
+	if thr := m.cfg.FeeLimitThreshold; thr != nil && thr.Cmp(bumpedFee) == 1 {
```

Comment:
Open UX question. We might consider multiplying the threshold with the `FeeLimitMultiplier` here, like
```suggestion
	if thr := m.cfg.FeeLimitThreshold; thr != nil && new(big.Int).Mult(thr, feeLimitMult).Cmp(bumpedFee) == 1 {
```

The reason is that if a limit of e.g. 1 Gwei is set, and we're starting from a very low fee, we wouldn't go over the configured threshold, but just stop short of increasing above it, if the next bump would go above the threshold. This might be unexpected.

Ex: If a threshold of 1 Gwei and limit-multiplier of 5 is set, I might expect fees to be bumped close to 5 Gwei, instead of close to the threshold itself of 1 Gwei.

Reply:
The threshold is already very configurable, I'd say we don't need to transform it more, and just document that it's applied at the exact value it's set to.

Reply:
Ok thanks, works for me. I realize that the current documentation already sounds like this is the case, so will leave as is.